storm-webapp/src/main/java/org/apache/storm/daemon/ui/resources/StormApiResource.java

```diff
@@ -0,0 +1,702 @@
+
+    public static Meter supervisorSummaryRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-supervisor-summary-http-requests");
+
+    public static Meter allTopologiesSummaryRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-all-topologies-summary-http-requests");
+
+    public static Meter topologyPageRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-topology-page-http-requests");
+
+    public static Meter topologyMetricRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-topology-metric-http-requests");
+
+    public static Meter buildVisualizationRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-build-visualization-http-requests");
+
+    public static Meter mkVisualizationDataRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-mk-visualization-data-http-requests");
+
+    public static Meter componentPageRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-component-page-http-requests");
+
+    public static Meter logConfigRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-log-config-http-requests");
+
+    public static Meter activateTopologyRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-activate-topology-http-requests");
+
+    public static Meter deactivateTopologyRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-deactivate-topology-http-requests");
+
+    public static Meter debugTopologyRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-debug-topology-http-requests");
+
+    public static Meter componentOpResponseRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-component-op-response-http-requests");
+
+    public static Meter topologyOpResponseMeter =
+            StormMetricsRegistry.registerMeter("ui:num-topology-op-response-http-requests");
+
+    public static Meter topologyLagRequestMeter =
+            StormMetricsRegistry.registerMeter("ui:num-topology-lag-http-requests");
+
+    public static Meter getOwnerResourceSummariesMeter =
+            StormMetricsRegistry.registerMeter("ui:num-get-owner-resource-summaries-http-request");
+
+
+    /**
+     * /api/v1/cluster/configuration -> nimbus configuration.
+     */
+
+    @GET
+    @Path("/cluster/configuration")
+    @Produces("application/json")
+    public Response getClusterConfiguration(@QueryParam("callback") String callback) throws TException {
+        clusterConfigurationRequestMeter.mark();
+        try (NimbusClient nimbusClient = NimbusClient.getConfiguredClient(config)) {
+            return UIHelpers.makeStandardResponse(
+                    nimbusClient.getClient().getNimbusConf(),
+                    callback, false
+            );
+        }
+    }
+
+    /**
+     * /api/v1/cluster/summary -> cluster summary.
+     */
+    @GET
+    @Path("/cluster/summary")
+    @NimbusOp("getClusterInfo")
```

Comment:
If the user is not allowed to do a `getClusterInfo` and callback is set will the resulting JSON from the mapped authorization exception have the callback in it or not?